src/redisraft.h

```diff
@@ -396,8 +397,11 @@ typedef struct RedisRaftCtx {
     struct ShardingInfo *sharding_info; /* Information about sharding, when cluster mode is enabled */
     RedisModuleDict *client_state;      /* A dict that tracks different client states */
     struct CommandSpecTable *commands_spec_table;
+
     /* Debug - Testing */
     struct RaftReq *debug_req;      /* Current RAFT.DEBUG request context, if processing one */
+    bool disable_snapshot;          /* If true, node will not take a snapshot */
```

Comment:
Hmm, let's think about that. Then, we'll add a config for each item, instead of subcommand. Not sure which one is better.

Reply:
First we can use a `debug` prefix, or just have a separate struct for debug config - so don't see any issue here. 
Right now we have:
*  RAFT.DEBUG DISABLE_SNAPSHOT <disable_snapshot> <disable_snapshot_load>
*  RAFT.DEBUG DISABLE_APPLY <val>
*  RAFT.DEBUG DELAY_APPLY <val>
*  RAFT.DEBUG MIGRATION_DEBUG [fail_connect|fail_import|fail_unlock|none]
All of them can be replaced by config command (actually, disable_apply is already a config). I think it will be simple, but it's up to you.

Reply:
No, my point is, `config get raft*` or probably `config rewrite` will list these debug configs. So, these debug configs will be listed among regular configs. Not sure if we care about this but Redis does not do it. There is a `debug` command, similar to `raft.debug`. 

I suggest we decide this later, not part of this PR.

Reply:
k, so in that case, let's continue with rafr.debug.